{"name": "Проверка логина c неправильным паролем", "status": "passed", "description": "Пытаемся залогиниться в аккаунт курьера, получаем ошибку 404 и соотвествующее сообщение", "start": 1718876272519, "stop": 1718876272660, "uuid": "4c2d43ed-1faa-4cc0-a39c-1ada3a339060", "historyId": "59c1190257832c75b2535cb60e758ffd", "testCaseId": "59c1190257832c75b2535cb60e758ffd", "fullName": "tests.test_courier_login.TestLoginCourier#test_no_such_username_and_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-02PLGO0"}, {"name": "thread", "value": "11616-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}